spec/lib/granite/rspec/raise_validation_error_spec.rb

```diff
@@ -29,27 +29,7 @@ def execute_perform!(*)
     let(:action) { Action.new(raise_error: true) }
 
     specify do
-      expect do
-        expect { action.perform! }.not_to raise_validation_error.of_type(:some_error)
-      end.to fail_with('expected not to raise validation error on attribute :base of type :some_error')
-    end
-
-    specify do
-      expect do
-        expect { action.perform! }.to raise_validation_error.of_type(:some_error2)
-      end.to fail_with('expected to raise validation error on attribute :base of type :some_error2, but raised {:base=>[{:error=>:some_error}]}')
-    end
-
-    specify do
-      expect do
-        expect { action.perform! }.to raise_validation_error.on_attribute(:raise_error)
-      end.to fail_with('expected to raise validation error on attribute :raise_error, but raised {:base=>[{:error=>:some_error}], :raise_error=>[]}')
-    end
-
-    specify do
-      expect do
-        expect { action.perform! }.to raise_validation_error.on_attribute(:raise_error).of_type(:some_error)
-      end.to fail_with('expected to raise validation error on attribute :raise_error of type :some_error, but raised {:base=>[{:error=>:some_error}], :raise_error=>[]}')
+      expect { action.perform! }.to raise_validation_error.on_attribute(:base).of_type(:some_error)
```

Comment:
I am not sure about the rationale behind those tests, it seems to me that a single expectation is enough to cover it.

Reply:
Nevermind, I restored them